tests/test_airflow.py

```diff
@@ -1,114 +0,0 @@
-import pathlib
-import subprocess
-
-import pytest
-
-from lineapy.api.api_utils import extract_taskgraph
-from lineapy.utils.utils import get_system_python_version, prettify
-
-
-def check_requirements_txt(t1: str, t2: str):
-    return set(t1.split("\n")) == set(t2.split("\n"))
-
-
-@pytest.mark.slow
-@pytest.mark.parametrize(
-    "artifact_names, dag_name, deps",
-    [
-        pytest.param(
-            ["p value"],
-            "sliced_housing_simple",
-            {},
-            id="sliced_housing_simple",
-        ),
-        pytest.param(
-            ["p value", "y"],
-            "sliced_housing_multiple",
-            {},
-            id="sliced_housing_multiple",
-        ),
-        pytest.param(
-            ["p value", "y"],
-            "sliced_housing_multiple_w_dependencies",
-            {"y": {"p value"}},
-            id="sliced_housing_multiple_w_dependencies",
-        ),
-    ],
-)
-def test_slice_airflow(artifact_names, dag_name, deps, airflow_plugin):
```

Comment:
I don't see an equivalent test for these in test_pipeline_run_airflow.py

Are we losing coverage? 
Is it because the underlying API has changed making these tests impossible to refactor?

Reply:
@andycui97, thanks for the astute question (as always!). The equivalent tests have been moved into `tests/unit/plugins/test_writers.py`, which is why you see several new tests cases being added in this PR. Hence, we have division of labor, where `test_writers.py` tests code refactor and `test_pipeline_run_airflow.py` tests whether the generated refactored files run successfully.

FYI, @lionsardesai @mingjerli

Reply:
sgtm, im not super familiar with our testing rn, just wanted to make sure